src/main/java/Duke.java

```diff
@@ -1,10 +1,30 @@
+import java.util.Scanner;
```

Comment:
Nice!! Imports are mostly very neat and it makes it readable! Good Job!!

Reply:
thank you